pgcli/pgclirc

```diff
@@ -31,6 +31,10 @@ history_file = default
 # and "DEBUG".
 log_level = INFO
 
+# Order of columns when expanding * to column list
+# Possible values: "table_order" and "alphabetic"
+column_order = table_order
```

Comment:
Let's make this something a little bit more specific like `asterisk_column_order`